lib/pagination/adapter.js

```diff
@@ -0,0 +1,58 @@
+const { invalidPageNumberError, invalidLimitNumberError } = require('./errors');
+
+let adapterParams = {};
```

Comment:
Like we talk some time ago, you should avoid this shared global variable, because wey don't like mutate state all the time and is not necessary here